### PR TITLE
Add discounts to the dashboard

### PR DESCRIPTION
**Pull request by davidfischer**
The discounts come in the form of Stripe coupons. The description can be edited in Stripe. Currently, we have just a 10% and 15% coupon based on the total campaign size.

They appear in:

- The create/edit flight screens (for staff). The discounts are not automatically applied yet. When a flight is renewed, the discount is renewed by default.
- They appear in the memo section of invoices
- They are displayed in the flight metadata to advertisers

## Screenshot

Screenshot from 2023-10-10 16-45-51

### CODE REVIEW

File: adserver/admin.py [hunk 1]
```diff
@@ -670,14 +670,14 @@ def action_create_draft_invoice(self, request, queryset):
             )
 
         # https://stripe.com/docs/api/invoices/create
+        description = "Thanks for your business!"
```
Comment (ericholscher) [inline adserver/admin.py:673]: Ha, I never noticed this before :)

**Review by ericholscher — APPROVED** [review]
Looks good.. So just to confirm, we aren't actually changing the pricing on the invoice, just noting the discount explicitly? That seems like a great first step. 👍

**Comment by davidfischer** [comment]
> Looks good.. So just to confirm, we aren't actually changing the pricing on the invoice, just noting the discount explicitly? That seems like a great first step. 👍

Correct. The pricing doesn't change at all. I did remove the weird logic we had where flights where priced per view if the number of views was not divisible by 1000 and CPM otherwise. It's always per view or click now.

File: adserver/admin.py (hunk 1 above)
Comment (davidfischer) [reply adserver/admin.py:673]: I usually fill out the memo line to reference a PO or if this is "invoice 1 of 3", etc.

**Pull request merged**